cogs5e/models/automation/effects/save.py

```diff
@@ -40,7 +40,15 @@ def run(self, autoctx):
         auto_pass = autoctx.args.last('pass', type_=bool, ephem=True)
         auto_fail = autoctx.args.last('fail', type_=bool, ephem=True)
         hide = autoctx.args.last('h', type_=bool)
-        adv = autoctx.args.adv(custom={'adv': 'sadv', 'dis': 'sdis'})
+
+        # check for combatant IEffects
+        if autoctx.combatant:
+            # Combine args/ieffect advantages - adv/dis (#1552)
+            adv = reconcile_adv(
+                adv= autoctx.args.last('sadv', bool, ephem=True) or autoctx.combatant.active_effects('adv'),
+                dis= autoctx.args.last('sdis', bool, ephem=True) or autoctx.combatant.active_effects('dis'))
```

Comment:
This doesn't seem to be checking that the save is for the correct ability, just that the argument in the ieffect exists (also, it's checking adv/dis instead of sadv/sdis)